client/settings/payments-and-transactions-section/__tests__/index.test.js

```diff
@@ -10,6 +11,10 @@ import {
 	useGetSavingError,
 } from 'wcstripe/data';
 
+jest.mock( '../../../data/account', () => ( {
+	useAccount: jest.fn(),
+} ) );
```

Comment:
These paths (and in the component module) can be made independent of nesting level, see for instance:
https://github.com/woocommerce/woocommerce-gateway-stripe/blob/13b254d7e16f5123c3e1110f48c78ddda586adab/client/components/payment-method-capability-status-pill/__tests__/payment-method-capability-status-pill.test.js#L4-L9

Reply:
Done in b106b0f.

Reply:
Thanks – not a big deal, but it's a similar case in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2263/commits/289285d5cdded32704b0999c7dac8db7e5211e24#diff-b6636d9b7c977e87e0fa162a64a0adb5a0cff40145488db12d4234f0e920ba03R5

Reply:
Done in 8408fcd.